sympy/stats/joint_rv_types.py

```diff
@@ -58,3 +58,154 @@ def marginal_distribution(self, indices, sym):
         return Lambda(sym, S(1)/sqrt((2*pi)**(len(_mu))*det(_sigma))*exp(
             -S(1)/2*(_mu - sym).transpose()*(_sigma**(-1)*\
                 (_mu - sym)))[0])
+
+#-------------------------------------------------------------------------------
+# Multivariate Laplace distribution ---------------------------------------------------------
+
+class MultivariateLaplaceDistribution(JointDistribution):
+    _argnames = ['mu', 'sigma']
+    is_Continuous=True
+
+    @property
+    def set(self):
+        k = len(self.mu)
+        return S.Reals**k
+
+    def check(self, mu, sigma):
+        mu, sigma = Matrix([mu]), Matrix(sigma)
+        _value_check(len(mu) == len(sigma.col(0)),
+            "Size of the mean vector and covariance matrix are incorrect.")
+        #check if covariance matrix is positive definite or not.
+        _value_check(all([i > 0 for i in sigma.eigenvals().keys()]),
+            "The covariance matrix must be positive definite. ")
+
+    def pdf(self, *args):
+        from sympy.functions.special.bessel import besselk
+        mu, sigma = Matrix(self.mu), Matrix(self.sigma)
```

Comment:
`Matrix` not needed here, `self.mu` and `self.sigma` are already supposed to be matrices.

Reply:
`self.sigma` and `self.mu` may also be of  type `list`, so `Matrix` just makes sure they are transformed into matrices.

Reply:
They should be stored as immutable matrices, the transformation should not happen here.